docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   web:
-    image: gpw-web
+    build: .
```

Comment:
Just for easy of development. Anyway you guys are building the gpw-web and using the local image.

This way, docker-compose will take care of building image and any change made to code base will be reflected automatically in code without need for any restart or image build.

If this is not needed, then this change can be reverted.

Reply:
The previous version used `build .`, but I remember it was changed to `image: gpw-web` for some reason, so it is better not to change.

Reply:
I've reverted the change and added a new compose to build and deploy. This can be used during development to prevent image rebuild everytime a change is made.

With this approach, and change made to the web files will be reflected automatically in the running instance without any restarts.